src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { SqlExtHostContext, SqlMainContext, ExtHostConnectionManagementShape, MainThreadConnectionManagementShape } from 'sql/workbench/api/node/sqlExtHost.protocol';
+import * as data from 'data';
+import { IExtHostContext } from 'vs/workbench/api/node/extHost.protocol';
+import { extHostNamedCustomer } from 'vs/workbench/api/electron-browser/extHostCustomers';
+import { IConnectionManagementService } from 'sql/parts/connection/common/connectionManagement';
+import { IObjectExplorerService } from 'sql/parts/registeredServer/common/objectExplorerService';
+import { IWorkbenchEditorService } from 'vs/workbench/services/editor/common/editorService';
+import * as TaskUtilities from 'sql/workbench/common/taskUtilities';
+import { IConnectionProfile } from 'sql/parts/connection/common/interfaces';
+import { dispose, IDisposable } from 'vs/base/common/lifecycle';
+
+@extHostNamedCustomer(SqlMainContext.MainThreadConnectionManagement)
+export class MainThreadConnectionManagement implements MainThreadConnectionManagementShape {
+
+	private _proxy: ExtHostConnectionManagementShape;
+	private _toDispose: IDisposable[];
+
+	constructor(
+		extHostContext: IExtHostContext,
+		@IConnectionManagementService private connectionManagementService: IConnectionManagementService,
```

Comment:
any reason not to prefix the private fields with '_'?  We seem to be doing that with the fields not defined in the constructor in this file.